src/store/flow/block-types/MobilePrimitives_SelectManyResponseBlockStore.ts

```diff
@@ -25,6 +26,44 @@ export const mutations: MutationTree<ICustomFlowState> = {
 export const actions: ActionTree<ICustomFlowState, IRootState> = {
   ...selectOneActions,
 
+  setMinChoices({commit, dispatch, rootGetters}, {blockId, value}: {blockId: IBlock['uuid'], value?: number}): void {
+    const block: ISelectManyResponseBlock = findBlockWith(blockId, rootGetters['flow/activeFlow']) as ISelectManyResponseBlock
+    commit('flow/block_updateConfigByKey', {
+      blockId: block.uuid,
+      key: 'minimum_choices',
+      value: isNumber(value) ? value : undefined,
+    }, {root: true})
+
+    const {vendor_metadata: metadata} = block as unknown as IBlockWithBranchingType
+    const {UNIFIED} = OutputBranchingType
+    if (metadata.io_viamo.branchingType === UNIFIED) {
+      dispatch('handleBranchingTypeChangedToUnified', {block})
+    }
+  },
+
+  setMaxChoices({commit, dispatch, rootGetters}, {blockId, value}: {blockId: IBlock['uuid'], value: number}): void {
+    const block: ISelectManyResponseBlock = findBlockWith(blockId, rootGetters['flow/activeFlow']) as ISelectManyResponseBlock
+    commit('flow/block_updateConfigByKey', {
+      blockId: block.uuid,
+      key: 'maximum_choices',
+      value: isNumber(value) ? value : undefined,
+    }, {root: true})
+
+    // todo: we should probably review our getters + setters in vue files and pull domain logic our stores ?! Schedule this?11
+    const {vendor_metadata: metadata} = block as unknown as IBlockWithBranchingType
+    const {UNIFIED} = OutputBranchingType
+    if (metadata.io_viamo.branchingType === UNIFIED) {
+      dispatch('handleBranchingTypeChangedToUnified', {block})
```

Comment:
Hi @safydy, I've opted to drop these in at the vuex layer. I believe that we should actually be constraining our domain logic at the vuex layer more often rather than complex-ifying vue component relationships by `$emit()`ting events— I think we're all guilty of this, including myself 😉. This way if anyone else tries to modify the property with `setMaxChoices()`, they'll also magically get their exits updated appropriately.

Do you happen to agree with this moving forward?

Reply:
@bzabos , great idea for moving it in vuex. Will keep in mind